mimisbrunnr.yaml

```diff
@@ -245,3 +245,12 @@ parserConfig:
         storePayload: true
         ruleTTL: 30s
         eventType: "State"
+
+
+########################################
+#   Filter Related Configuration
+########################################
+filterConfig:
```

Comment:
I don't think we should have "Config" in the yaml (since everything is config).  But I'm making a separate issue for this, so that change can be done later if you want.